usb_cameras:
  front:
    device_path: /dev/video_front
    topic: /camera/usb/front/compressed
    horizontal_fov: 82
    focal_length: 2.65
    sensor_size:
      width: 3.054
      height: 1.718
    img_size:
      width: 640
      height: 480
    frame_id: camera_usb_front

  bottom:
    device_path: /dev/video_bottom
    topic: /camera/usb/bottom/compressed
    horizontal_fov: 82
    focal_length: 2.65
    sensor_size:
      width: 3.054
      height: 1.718
    img_size:
      width: 640
      height: 480
    frame_id: camera_usb_bottom